docs/departments/security_department/installation_defense.md

```diff
@@ -17,8 +17,6 @@ Security personnel guarding the Main Guard Line should only be focused on Class
 
 If the glass is broken in the observation windows, you are required to call in a member of the Engineering & Technical Services department.
 
-If you found yourself still confused by said positions. This link will provide you a much more physical interpretation plus more positions not just in the inner CDC itself.
-
 ### Solitary Usage
 
 Solitary Usage
```

Comment:
Fix formatting by removing spaces after the asterisks and before Instant Solitary. (** Instant Solitary**)